src/java/org/apache/cassandra/tools/LoaderOptions.java

```diff
@@ -653,10 +653,10 @@ private static CmdLineOptions getCmdLineOptions()
         options.addOption("p",  NATIVE_PORT_OPTION, "native transport port", "port used for native connection (default 9042)");
         options.addOption("sp",  STORAGE_PORT_OPTION, "storage port", "port used for internode communication (default 7000)");
         options.addOption("ssp",  SSL_STORAGE_PORT_OPTION, "ssl storage port", "port used for TLS internode communication (default 7001)");
-        options.addOption("t", THROTTLE_MBITS, "throttle", "throttle speed in Mbits (default unlimited)");
-        options.addOption("idct", INTER_DC_THROTTLE_MBITS, "inter-dc-throttle", "inter-datacenter throttle speed in Mbits (default unlimited)");
-        options.addOption("e", ENTIRE_SSTABLE_THROTTLE_MBITS, "entire-sstable-throttle", "entire SSTable throttle speed in Mbits (default unlimited)");
-        options.addOption("eidct", ENTIRE_SSTABLE_INTER_DC_THROTTLE_MBITS, "entire-sstable-inter-dc-throttle", "entire SSTable inter-datacenter throttle speed in Mbits (default unlimited)");
+        options.addOption("t", THROTTLE_MEGABITS, "throttle", "throttle speed in Mbps (default unlimited)");
+        options.addOption("idct", INTER_DC_THROTTLE_MEGABITS, "inter-dc-throttle", "inter-datacenter throttle speed in Mbps (default unlimited)");
+        options.addOption("e", ENTIRE_SSTABLE_THROTTLE_MEBIBYTES, "entire-sstable-throttle", "entire SSTable throttle speed in MiB/s (default unlimited)");
+        options.addOption("eidct", ENTIRE_SSTABLE_INTER_DC_THROTTLE_MEBIBYTES, "entire-sstable-inter-dc-throttle", "entire SSTable inter-datacenter throttle speed in MiB/s (default unlimited)");
```

Comment:
Note for the second reviewer: we were discussing with Francisco that it would have been nice if we had all 4 parameters with the same unit. Probably we can add a note that those in bits are deprecated(but still keep them for compatibility) and add two more for them which can be used for the mebibytes/s. On the other hand we are already in a freeze so probably the best is to leave things as is now and add for the next release in trunk directly the opportunity to set them with units, similar to the cassandra.yaml

Reply:
In this approach, would it make sense for all four options has the same unit as MEGABITS, or declare new names for the ones using MEBIBYTES? I'd prefer the later one. 
The deprecation approach requires to declare _new_ options and mark the old ones as deprecated. Both new and old options are present. Currently, the option names are consistent with different unit. When the new option is added, there will be options names that are not consistent, but the unit is. Let's say, the new option name for throttle are `tmib` and `throttle-mib` (shorthand and full accordingly). `tmib` and `e` has inconsistent name, but they use the same unit.

Reply:
The deprecation approach suggested by @yifan-c  makes sense to me. 
I would expect if someone wants to use the 4 of them to use the new ones mebibytes/s for all of them.
The old two properties with megabits/s are for compatibility with scripts. If someone wants to add also the other 2 they will decide whether they want to migrate the old ones to the new units or just add the two new properties with different units and be inconsistent.

Reply:
So with Yifan's suggestion we'll have the following:
`t`, `throttle` -> megabits (deprecated)
`tmib`, `throttle-mib` -> mebibytes (new)
`emib`, `entire-sstable-throttle-mib` -> mebibytes (new)
`idct`, `inter-dc-throttle` -> megabits (deprecated)
`idctmib`, `inter-dc-throttle-mib` -> mebibytes (new)
`eidctmib`, `entire-sstable-inter-dc-throttle-mib` -> mebibytes (new)

What do you think @yifan-c, @ekaterinadimitrova2 ?

Reply:
Looks good to me.

Reply:
`idctib` and `eidctmib` sound a bit long but they follow the already established naming convention and when people know where it comes from I guess it will be intuitive.

Reply:
I pushed the changes above. Let me know what you think